plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -519,6 +520,26 @@ public void testUnsupportedColumnType()
         session.execute("DROP KEYSPACE keyspace_6");
     }
 
+    @Test
+    public void testNullAndEmptyTimestamp()
+    {
+        String keyspace = "keyspace_7";
+        String tableName = "test_empty_timestamp";
+
+        session.execute("DROP KEYSPACE IF EXISTS " + keyspace);
+        session.execute(format("CREATE KEYSPACE %s WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor': 1}", keyspace));
+        session.execute(format("CREATE TABLE %s.%s (id int primary key, column_1 timestamp, column_2 timestamp)", keyspace, tableName));
```

Comment:
Would it make sense to test with timestamp column being primary key and one that is not?

Reply:
Cassandra primary key doesn't allow null or empty timestamp, so we don't have to (≒ can't) add such test.

Reply:
And clustering keys?

Reply:
Partition and clustering keys doesn't allow null, but it allows empty timestamp. Let me add some tests.